blog/steering/governance-changes.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: 'Changes to the Knative Project Governance'
+Author: "[Paul Morie](https://github.com/pmorie)", on behalf of the [Knative Steering Committee](https://github.com/knative/community/blob/master/STEERING-COMMITTEE.md)
+Author handle: https://github.com/pmorie
+Description: Summary of governance changes to steering and trademark committees.
+Folder with media files: 'N/A'
+Blog URL: ''
+Labels: Articles
+Reviewers: ''
+date: 2020-09-30
+description: "Knative Project Governance Changes"
+type: "blog"
+
+---
+| Reviewer  | Date | Approval |
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here --> |   |  |
+
+
+One of the long-standing [asks](https://github.com/knative/community/issues/33)
+within the Knative community has been for the Steering Committee to clarify its
+charter and how the committee is composed. I'm happy to be able to say that we
+now have a new [Steering Committee
+charter](https://github.com/knative/community/pull/277) which, in my opinion,
+delivers on this ask. Community participation was central to working through the
+process to get to this new charter, and I first and foremost want to say a
+sincere THANK YOU to everyone in our community that participated in our public
```

Comment:
should we have links [pulbic discussions] to the PRs and video recordings of the meetings for transparency?